kustomize/components/redis-enterprise/kustomization.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+patchesStrategicMerge:
+# cartservice - replace REDIS_ADDR to target new fully managed Redis Enterprise database instance
+- |-
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: cartservice
+  spec:
+    template:
+      spec:
+        containers:
+          - name: server
+            env:
+            - name: REDIS_ADDR
+              value: "{{REDIS_ADDR}}"
```

Comment:
This `{{REDIS_ADDR}}` is now renamed in `main` as `REDIS_CONNECTION_STRING`, could you please update it accordingly here too please?

Reply:
Re1: We do not have documentation for how to deploy our Google Marketplace Redis Enterprise solution.  Google Cloud team did create a video - https://www.youtube.com/watch?v=GvnmAAPT9aw but for our outdated listing.  Once the new listing video is available, we can replace most of the screenshots then.

Reply:
Could we have an official public doc in our docs about that? Or at least a blog article somewhere like for example here: https://medium.com/google-cloud?

The intent in this repo is to have pointers to the setup but not that much detailed step-by-step instructions. As we will scale with these important end-to-end scenarios, it will be hard to maintain, keep up to date, etc.

Reply:
Just committed the changes to update REDIS_CONNECTION_STRING.  

For step-by-step instructions, what do you think if I move this https://github.com/gmflau/microservices-demo/blob/main/kustomize/components/redis-enterprise/redis-enterprise.md page to our https://github.com/Redislabs-Solution-Architects/ repo so I could maintain it outside of your https://github.com/GoogleCloudPlatform/microservices-demo repo?

Reply:
Oh ok, I just looked at https://github.com/Redislabs-Solution-Architects, you actually have all of that in there (doc and terraform), impressive!

For the doc, I think that makes sense to point to your repo from here.

For Terraform, I'm wondering, should we take the same approach? Or having this here for that part only? CC: @NimJay

Reply:
Moved step-by-step to https://github.com/Redislabs-Solution-Architects/redis-enterprise-cloud-gcp/blob/main/marketplace/gcp/redis-enterprise.md